From 5043cbcaaca10209b324f6aa7d06bc6b2f3b6ece Mon Sep 17 00:00:00 2001
From: Victor Gu <xigu@marvell.com>
Date: Fri, 4 Jul 2014 15:30:13 +0800
Subject: [PATCH 1770/1825] fix: slic: update lantiq BBD file for AC and DC
 attenuation

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit c28d32d986643a9265d4f38c2c04281149f60b4b

    Change AC and DC attenuation to 0 for Lantiq DFEV SLIC.
Signed-off-by: Victor Gu <xigu@marvell.com>

Change-Id: I323a1802a103c988a0f0217237ef21177874e061
Reviewed-on: http://vgitil04.il.marvell.com:8080/8964
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Yehuda Yitschak <yehuday@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 tools/voice/lantiq-dfev/libtapi-fw.c |   59 ++++++++++++++-------------------
 1 files changed, 25 insertions(+), 34 deletions(-)

diff --git a/tools/voice/lantiq-dfev/libtapi-fw.c b/tools/voice/lantiq-dfev/libtapi-fw.c
index 2fce16f..c7b94c2 100644
--- a/tools/voice/lantiq-dfev/libtapi-fw.c
+++ b/tools/voice/lantiq-dfev/libtapi-fw.c
@@ -64,40 +64,31 @@
 
 unsigned char tapi_bbd_blob[] = {
 	/* Source: R600.BIN */
-	0x00, 0x0A, 0x00, 0x01, 0x00, 0x00, 0x00, 0x0C, 0x21, 0x62, 0x62, 0x64,
-	0x4D, 0x41, 0x52, 0x56, 0x0D, 0x07, 0x0A, 0xFF, 0x10, 0x01, 0x00, 0x01,
-	0x00, 0x00, 0x01, 0x08, 0x00, 0x00, 0x80, 0xE7, 0x31, 0xE1, 0x6F, 0xE8,
-	0x34, 0x45, 0xDA, 0x40, 0xBB, 0xD9, 0xEA, 0x68, 0x2A, 0x57, 0x2C, 0x9A,
-	0xDA, 0x8D, 0xF2, 0xF2, 0x4C, 0x9F, 0xEA, 0x2D, 0x40, 0x00, 0x7F, 0xFF,
-	0xF2, 0x5B, 0x02, 0x07, 0xFF, 0xA7, 0xFF, 0xDC, 0xFF, 0x84, 0x00, 0x00,
-	0x00, 0x8A, 0xFF, 0x5C, 0x00, 0x7C, 0xFF, 0xC7, 0x22, 0x12, 0xE8, 0x41,
-	0x33, 0xD7, 0xC5, 0x30, 0x20, 0xFF, 0x99, 0xA7, 0x16, 0x98, 0x3B, 0x70,
-	0xD0, 0x84, 0x46, 0xD1, 0xAC, 0x0C, 0x24, 0x4E, 0x8D, 0xA3, 0x14, 0xD5,
-	0x5E, 0x22, 0xB3, 0xC4, 0x73, 0x20, 0xA2, 0x69, 0x65, 0x84, 0x8C, 0x54,
-	0x5C, 0x67, 0x6E, 0x42, 0x8D, 0xD8, 0x7E, 0xE9, 0x7F, 0x0E, 0x80, 0xBB,
-	0x7F, 0xFF, 0x7F, 0xF4, 0x37, 0xED, 0x28, 0xE4, 0x7F, 0xFF, 0x20, 0x00,
-	0x7F, 0xFF, 0xFA, 0xF1, 0xFE, 0x7F, 0x01, 0x66, 0xFD, 0xB2, 0x00, 0xBD,
-	0xFF, 0x43, 0x00, 0xF4, 0xFF, 0x26, 0x00, 0x92, 0xFF, 0xBF, 0x24, 0xF1,
-	0xE8, 0x9D, 0x37, 0x7C, 0xC8, 0x52, 0x21, 0xC4, 0x9B, 0xFA, 0x14, 0xF2,
-	0x3B, 0x70, 0xD0, 0x84, 0x46, 0xD1, 0xAC, 0x0C, 0x24, 0x4E, 0x8D, 0xA3,
-	0x14, 0xD5, 0x5D, 0x96, 0xB3, 0xFD, 0x72, 0xC6, 0xA1, 0xD6, 0x65, 0x42,
-	0x8B, 0xE5, 0x5C, 0x7A, 0x7B, 0x75, 0x7F, 0x9E, 0x25, 0xF3, 0x4A, 0xC8,
-	0x7F, 0xFF, 0xFD, 0xF9, 0x2C, 0x2F, 0x1B, 0x04, 0xF2, 0x1D, 0x08, 0x13,
-	0xFA, 0x2D, 0x03, 0xD9, 0xFD, 0x03, 0x02, 0x02, 0xFE, 0x25, 0x01, 0x4E,
-	0x7F, 0xCF, 0x20, 0x00, 0xDB, 0x06, 0x4B, 0x72, 0xD0, 0x1C, 0x39, 0x99,
-	0x0A, 0x3D, 0x09, 0x21, 0x00, 0x00, 0x00, 0x00, 0x00, 0x08, 0x00, 0xC9,
-	0x00, 0x06, 0x00, 0x00, 0x00, 0x10, 0x00, 0x00, 0x00, 0x18, 0x00, 0x00,
-	0x2D, 0xE4, 0x7C, 0xF4, 0x3B, 0x6F, 0x1A, 0xD6, 0xC4, 0x5F, 0x12, 0x0B,
-	0x0B, 0x85, 0x00, 0x00, 0x12, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x18,
-	0x04, 0x00, 0x05, 0x14, 0x04, 0x00, 0x03, 0x33, 0x3F, 0x1C, 0x06, 0x66,
-	0x0A, 0xAA, 0x48, 0xB4, 0x2E, 0xE0, 0x37, 0x89, 0x00, 0x00, 0x4A, 0x3D,
-	0x12, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x1C, 0x04, 0x00, 0x04, 0x18,
-	0x99, 0x90, 0xB8, 0x28, 0xF7, 0xC8, 0x00, 0xC8, 0x17, 0x0A, 0x0A, 0x3D,
-	0x17, 0x0A, 0x07, 0xAE, 0x23, 0x8E, 0x19, 0x99, 0x7E, 0xB8, 0x26, 0x66,
-	0x12, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x0C, 0x04, 0x00, 0x06, 0x08,
-	0x55, 0x55, 0x33, 0x33, 0x00, 0x1B, 0x14, 0x28, 0x12, 0x00, 0x00, 0x01,
-	0x00, 0x00, 0x00, 0x0C, 0x04, 0x00, 0x0C, 0x08, 0x00, 0x14, 0x00, 0xFF,
-	0xDA, 0x33, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+	0x00, 0x0A, 0x00, 0x01, 0x00, 0x00, 0x00, 0x0C, 0x21, 0x62, 0x62, 0x64, 0x4D, 0x41, 0x52, 0x56,
+	0x0E, 0x07, 0x04, 0xFF, 0x10, 0x01, 0x00, 0x01, 0x00, 0x00, 0x01, 0x08, 0x00, 0x00, 0x80, 0xE7,
+	0x31, 0xE1, 0x6F, 0xE8, 0x34, 0x45, 0xDA, 0x40, 0xBB, 0xD9, 0xEA, 0x68, 0x2A, 0x57, 0x2C, 0x9A,
+	0xDA, 0x8D, 0xF2, 0xF2, 0x4C, 0x9F, 0xEA, 0x2D, 0x40, 0x00, 0x7F, 0xFF, 0xF2, 0x5B, 0x02, 0x07,
+	0xFF, 0xA7, 0xFF, 0xDC, 0xFF, 0x84, 0x00, 0x00, 0x00, 0x8A, 0xFF, 0x5C, 0x00, 0x7C, 0xFF, 0xC7,
+	0x22, 0x12, 0xE8, 0x41, 0x33, 0xD7, 0xC5, 0x30, 0x20, 0xFF, 0x99, 0xA7, 0x16, 0x98, 0x3B, 0x70,
+	0xD0, 0x84, 0x46, 0xD1, 0xAC, 0x0C, 0x24, 0x4E, 0x8D, 0xA3, 0x14, 0xD5, 0x5E, 0x22, 0xB3, 0xC4,
+	0x73, 0x20, 0xA2, 0x69, 0x65, 0x84, 0x8C, 0x54, 0x5C, 0x67, 0x6E, 0x42, 0x8D, 0xD8, 0x7E, 0xE9,
+	0x7F, 0x0E, 0x80, 0xBB, 0x7F, 0xFF, 0x7F, 0xF4, 0x4F, 0x00, 0x28, 0xE3, 0x7F, 0xFF, 0x20, 0x00,
+	0x7F, 0xFF, 0xFA, 0xF1, 0xFE, 0x7F, 0x01, 0x66, 0xFD, 0xB2, 0x00, 0xBD, 0xFF, 0x43, 0x00, 0xF4,
+	0xFF, 0x26, 0x00, 0x92, 0xFF, 0xBF, 0x24, 0xF1, 0xE8, 0x9D, 0x37, 0x7C, 0xC8, 0x52, 0x21, 0xC4,
+	0x9B, 0xFA, 0x14, 0xF2, 0x3B, 0x70, 0xD0, 0x84, 0x46, 0xD1, 0xAC, 0x0C, 0x24, 0x4E, 0x8D, 0xA3,
+	0x14, 0xD5, 0x5D, 0x96, 0xB3, 0xFD, 0x72, 0xC6, 0xA1, 0xD6, 0x65, 0x42, 0x8B, 0xE5, 0x5C, 0x7A,
+	0x7B, 0x75, 0x7F, 0x9E, 0x78, 0x00, 0x4A, 0xC9, 0x7F, 0xFF, 0xFD, 0xF9, 0x2C, 0x2E, 0x1B, 0x03,
+	0xF2, 0x1D, 0x08, 0x13, 0xFA, 0x2D, 0x03, 0xD9, 0xFD, 0x03, 0x02, 0x02, 0xFE, 0x25, 0x01, 0x4E,
+	0x7F, 0xCF, 0x20, 0x00, 0xDB, 0x06, 0x4B, 0x72, 0xD0, 0x1C, 0x39, 0x99, 0x0A, 0x3D, 0x09, 0x21,
+	0x00, 0x00, 0x00, 0x00, 0x00, 0x08, 0x00, 0xC9, 0x00, 0x06, 0x00, 0x00, 0x00, 0x10, 0x00, 0x00,
+	0x00, 0x18, 0x00, 0x00, 0x2D, 0xE4, 0x7C, 0xF4, 0x3B, 0x6F, 0x1A, 0xD6, 0xC4, 0x5F, 0x12, 0x0B,
+	0x09, 0x37, 0x00, 0x00, 0x12, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x18, 0x04, 0x00, 0x05, 0x14,
+	0x14, 0x00, 0x03, 0x33, 0x4B, 0xBB, 0x48, 0xB4, 0x0A, 0xAA, 0x48, 0xB4, 0x2E, 0xE0, 0x38, 0xCC,
+	0x00, 0x00, 0x4A, 0x3D, 0x12, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x1C, 0x04, 0x00, 0x04, 0x18,
+	0x99, 0x90, 0xB8, 0x08, 0x07, 0xC8, 0x00, 0xC8, 0x0C, 0xCC, 0x0A, 0x3D, 0x0C, 0xCC, 0x07, 0xAE,
+	0x2A, 0xAA, 0x20, 0x00, 0x7E, 0xB8, 0x26, 0x66, 0x12, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x0C,
+	0x04, 0x00, 0x0C, 0x08, 0x00, 0x14, 0x00, 0xFF, 0xB3, 0x33, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+	0x00, 0x00, 0x00, 0x00
 };
 
 unsigned int tapi_bbd_blob_size = sizeof(tapi_bbd_blob);
-- 
1.7.5.4

